#!/usr/bin/env python
PACKAGE = "reconfigurable_hippo"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("k_11", double_t, 0, "k_11 (position)", 1.0, 0, 10)
gen.add("k_12", double_t, 0, "k_12 (orientation)", 1.0, 0 , 10)
gen.add("k_2", double_t, 0, "k_2", 1.0, 0, 10)
gen.add("k_3", double_t, 0, "k_3", 1.0, 0, 10)
gen.add("K_1", double_t, 0, "K_1", 1.0, 0, 10)
gen.add("Khat_1", double_t, 0, "Khat_1", 1.0, 0, 10)
gen.add("K_2", double_t, 0, "K_2", 1.0, 0, 10)
gen.add("Khat_2", double_t, 0, "Khat_2", 1.0, 0, 10)
gen.add("K_3", double_t, 0, "K_3", 1.0, 0, 10)
gen.add("Khat_3", double_t, 0, "Khat_3", 1.0, 0, 10)

exit(gen.generate(PACKAGE, "controller_node", "SlidingModeController"))
